README.md

```diff
@@ -31,9 +31,11 @@ Build Requirements
 Building oreboot
 -----------------
 
+To setup your Rust environment for oreboot, see below:
+
 ```
-# set up the environment
-source ENV
+# Install some tools we will need later
+sudo apt install device-tree-compiler pkg-config
```

Comment:
why did you remove 
source ENV
? I use it.

Reply:
Ah, I thought it was an old file - it seems it's a new file (my branch didn't have it).
I'll re-add it. I didn't have the targets/ directory either.